Solutions/Corvus.Json.Specs/Features/JsonModel/JsonSerializationWithSerializerDisabled.feature

```diff
@@ -0,0 +1,341 @@
+	Writing entities
+
+Scenario Outline: Serialize a jsonelement-backed JsonAny to a string
+	Given the JsonElement backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization without enabling inefficient serialization
+	Then the deserialization operation should throw an InvalidOperationException
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+Scenario Outline: Serialize a dotnet-backed JsonAny to a string
+	Given the dotnet backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization without enabling inefficient serialization
+	Then the deserialization operation should throw an InvalidOperationException
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+
+Scenario Outline: Write a jsonelement-backed JsonAny to a string
+	Given the JsonElement backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization without enabling inefficient serialization
+	Then the deserialization operation should throw an InvalidOperationException
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+Scenario Outline: Write a dotnet-backed JsonAny to a string
+	Given the dotnet backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization without enabling inefficient serialization
+	Then the deserialization operation should throw an InvalidOperationException
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+Scenario Outline: Write a jsonelement-backed JsonNotAny to a string
```

Comment:
Again, this looks like a big long list of near-identical scenarios that you were about to turn into a single outline but then got distracted.